src/modules/claim/ticket/ClaimTicketForm.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, VStack } from '@chakra-ui/react';
+import { useFormContext } from 'react-hook-form';
+
+import { useClaimForm } from '../ClaimFormContext';
+
+import { NameField } from './fields/NameField';
+import { EmailField } from './fields/EmailField';
+
+export interface ClaimTicketFormFieldTypes {
+  name: string;
+  email: string;
+}
+
+interface ClaimTicketFormProps {}
+
+export const ClaimTicketForm = ({}: ClaimTicketFormProps) => {
+  const { getClaimFormData } = useClaimForm();
+  const { handleSubmit, control } = useFormContext<ClaimTicketFormFieldTypes>();
+
+  const handleSubmitClick = () => {
+    return console.log(getClaimFormData());
```

Comment:
What's the purpose of `getClaimFormData` and returning `console.log` here? I think we can have a `handleSubmit` function in Context? similar to `handleDropConfirmation`

Reply:
the `console.log` is only a placeholder
the `handleSubmitClick` is in case there needs to be more UI changes when clicking, like maybe `loading` state or some other states.